docs/fips.md

```diff
@@ -0,0 +1,35 @@
+# FIPS compliance
+
+The Federal Information Processing Standards (FIPS) are a set of U.S. government standards that ensure the security of computer systems for non-military government agencies and contractors. These standards specify how to perform cryptographic operations, such as encryption, hashing, and digital signatures. FIPS mode is a mode of operation that enforces these standards and rejects any non-compliant algorithms or parameters.
+
+Percona Server for MySQL implements the same level of FIPS support as MySQL. Percona Server for MySQL can run in FIPS mode if a FIPS-enabled OpenSSL library and FIPS Object Module are available at runtime or if compiled using a compliant version of OpenSSL. You can also receive this functionality by [building Percona Server for MySQL from source code](compile-percona-server.md).
+
+Percona Server for MySQL uses the same variables and values as MySQL. Percona Server for MySQL enables control of FIPS mode on the server side and the client side:
+
+* The `ssl_fips_mode` system variable controls whether the server operates in FIPS mode. This variable is disabled by default.
+
+    The `ssl_fips_mode` system variable has these values:
+
+    * `0` - disables FIPS mode
+    * `1` - enables FIPS mode
+    * `2` - enables `strict` FIPS mode
+
+* The `--ssl-fips-mode` client option controls whether a given MySQL client operates in FIPS mode. This setting does not change the server setting. This option is disabled by default.
+
+    The `--ssl-fips-mode` client option has these values:
+
+    * `OFF` - disables FIPS mode
+    * `ON` - enables FIPS mode
+    * `STRICT` - enables `strict` FIPS mode. This value provides more restrictions than the `ON` value. However, MySQL itself does not have any FIPS-specific code other than to specify the FIPS mode value to OpenSSL. The exact behavior of FIPS mode for `ON` or `STRICT` depends on the OpenSSL version.
+     
+    Note that the MySQL server does not read the OpenSSL configuration file unless the `--ssl-fips-mode` client option is set to `ON` or `STRICT`. The `--ssl-fips-mode` client option is read-only and cannot be changed at runtime.
+  
+    To enable the FIPS mode, pass `--ssl-fips-mode=ON` to mysqld as a command line argument. Ignore the warning that the `--ssl-fips-mode` client option is deprecated. To ensure that the FIPS mode is enabled, check that the error log contains the following message:
```

Comment:
Make this a separate section

## Check FIPS Mode

Reply:
updated